File: src/main/java/de/fxnm/web/components/submission/success/CheckOutputLineData.java
package de.fxnm.web.components.submission.success;

import lombok.Getter;

public class CheckOutputLineData {

    @Getter
    private final String type;
    @Getter
    private final String content;

    public CheckOutputLineData(final String type, final String content) {
        this.type = type;
        this.content = content;
    }

    @Override
    public String toString() {
        return String.format("%-10s >%s", this.type, this.content.replace(" ", "␣"));
    }
}

File: src/main/java/de/fxnm/web/components/submission/success/CheckOutputLineData.kt
package de.fxnm.web.components.submission.success

class CheckOutputLineData(val type: String, private val content: String) {

    fun getContent(): String {
        return if (type == "ERROR" || type == "OTHER") {
            content
        } else {
            content.replace(" ", "␣")
        }
    }
}
